WordPress/src/main/res/values/strings.xml

```diff
@@ -3636,12 +3632,14 @@
     <string name="storage_utils_dialog_ok_button">View Storage</string>
     <string name="storage_utils_dialog_dont_show_button">Don\'t Show Again</string>
 
-    <!-- Blogging reminders -->
-    <string name="set_your_blogging_goals_title">Set your blogging goals</string>
-    <string name="set_your_blogging_goals_message">Your post is publishing… in the meantime, set up your blogging goals to get reminders, and track your progress.</string>
-    <string name="set_your_blogging_goals_button">Set goals</string>
-    <string name="blogging_goals_n_a_week">%s a week</string>
-    <string-array name="blogging_goals_count" translatable="false">
+    <!-- Blogging Reminders -->
+    <string name="blogging_reminders_notification_title">%s, it\'s time to blog</string>
+    <string name="blogging_reminders_notification_text">This is your reminder to blog today &#9997;</string>
+    <string name="set_your_blogging_reminders_title">Set your blogging reminders</string>
+    <string name="set_your_blogging_reminders_message">Your post is publishing… in the meantime, set up your blogging goals to get reminders, and track your progress.</string>
```

Comment:
I think this is on purpose but just to be sure, we still want the word "goals" in here, right?

Reply:
We actually wanted to update this as well but simply replacing "goals" with "reminders" wouldn't work here as the phrase would get redundant. 

I brought this up for discussion (internal ref: p1624445050255400-slack-C01UWB9GD1Q) and was waiting for a decision to be made before I could take this out of draft. 

I've updated the string on f81d853a and this is now ready for review.